lms/templates/help_modal.html

```diff
@@ -40,81 +40,88 @@ <h2>
 
-<%
-  discussion_tab = CourseTabList.get_discussion(course) if course else None
-  discussion_link = discussion_tab.link_func(course, reverse) if (discussion_tab and discussion_tab.is_enabled(course, user=user)) else None
-  has_user_registered = UserProfile.has_registered(user)
-%>
+    <%
+      discussion_tab = CourseTabList.get_discussion(course) if course else None
+      discussion_link = discussion_tab.link_func(course, reverse) if (discussion_tab and discussion_tab.is_enabled(course, user=user)) else None
+      has_user_registered = UserProfile.has_registered(user)
+    %>
 
-% if discussion_link and has_user_registered:
-    <p>${Text(_(
-      'For {strong_start}questions on course lectures, homework, tools, or materials for this course{strong_end}, post in the course discussion forum.'
-    )).format(
-      strong_start=HTML('<strong>'),
-      strong_end=HTML('</strong>')
-    )}</p>
-    <ul class="help-buttons">
-      <li><a class="help-modal-button"  href="${discussion_link}">
-        ${_('Post in the Forum')}
-      </a></li>
-    </ul>
-% endif
+    % if discussion_link and has_user_registered:
+        <p>${Text(_('For {strong_start}questions on course lectures, homework, tools, or materials for this course{strong_end}, post in the {link_start}course discussion forum{link_end}.')).format(
+          strong_start=HTML('<strong>'),
+          strong_end=HTML('</strong>'),
+          link_start=HTML('<a href="{url}" target="_blank">').format(
+            url=discussion_link
+          ),
+          link_end=HTML('</a>'),
+          )}
+        </p>
+    % endif
 
-    <p>${Text(_(
-      'Have {strong_start}general questions about {platform_name}{strong_end}? You can find lots of helpful information in the {platform_name} Help Center.'
-    )).format(
-      strong_start=HTML('<strong>'),
-      strong_end=HTML('</strong>'),
-      platform_name=static.get_platform_name()
-    )}</p>
+        <p>${Text(_('Have {strong_start}general questions about {platform_name}{strong_end}? You can find lots of helpful information in the {platform_name} {link_start}Help Center{link_end}.')).format(
+            strong_start=HTML('<strong>'),
+            strong_end=HTML('</strong>'),
+            link_start=HTML('<a href="{url}" id="feedback-faq-link" target="_blank">').format(
+              url=marketing_link('FAQ')
+            ),
+            link_end=HTML('</a>'),
+            platform_name=static.get_platform_name())}
+          </p>
 
-    <ul class="help-buttons">
-      <%include file="help-center-link.html" />
-    </ul>
+        <div class="help-buttons">
+          <li>
+              <a class="help-modal-button" href="${marketing_link('FAQ')}" target="_blank">
+                  ${_('Access the Help Center')}
+              </a>
+          </li>
+        </div>
+        <hr>
 
-    <hr>
-% if course:
-    <%include file="help-contact-link.html" />
+    % if course:
+        <%include file="help-contact-link.html" />
```

Comment:
can't remove this, we need to override both `href` and `id` for the link, so that when `id` is set JS uses it, and when `href` is set it behaves like a link...

Reply:
Can't we just always include the same ID?

Reply:
no because the JS is gonna always look for that ID and `preventDefault` if it finds it...